### ISSUE TITLE
Euronext historical data bugs?

### ISSUE THREAD

**Issue by piacimosca** [antoinevulcain/Financial-Modeling-Prep-API#49]
getting 5m historical data for: 
OR.PA gives values in time range: 12:05 -> 15:40 (and no data between 12:50 and 15:05)
BIM.PA the range is: 09:05 -> 17:40 (and no data between 13:20 and 15:05)

to name a few. what seems to be true for all (or at least that many I've tried) euronext quotes, is that there is a data drop, and always seems to end at 15:05
time range varies but 12:05 -> 15:40 showed up quite often. lisbon, paris, etc the same.

**Comment by antoinevulcain** [comment]
Yes that's normal it's for all Euronext I did some settings with cp30 sorry about that 
Will be all good and running by tomorrow

**Comment by piacimosca** [comment]
oh, at least you don't have to investigate. 
thanks!

**Comment by antoinevulcain** [comment]
Yes that's rare this time :)

**Comment by piacimosca** [comment]
is there something similar with other exchanges?

[lhttps://financialmodelingprep.com/api/v3/historical-chart/5min/AAPL](https://financialmodelingprep.com/api/v3/historical-chart/5min/AAPL)

may 4th starts at 9:30
may 1st at 9:40
april 30th at 9:45
april 29th at 9:30 again

**Comment by antoinevulcain** [comment]
Yes this week has been a lot of implementations but now it is all set and ready. As time flys data will adjust
